.env.sample

```diff
@@ -1,4 +1,6 @@
-POSTGRES_HOST=POSTGRES_HOST
-POSTGRES_DB=POSTGRES_DB
-POSTGRES_USER=POSTGRES_USER
-POSTGRES_PASSWORD=POSTGRES_PASSWORD
\ No newline at end of file
+POSTGRES_HOST=
+POSTGRES_DB=
+POSTGRES_USER=
+POSTGRES_PASSWORD=
+POSTGRES_PORT=
+PGDATA=
```

Comment:
It's fine to add examples to your variables (like `POSTGRES_HOST=POSTGRES_HOST`), I meant to add them to each variable in this file!